sdk/Lusid.Sdk.Tests/tutorials/Ibor/Orders.cs

```diff
@@ -356,5 +357,117 @@ public void Upsert_And_Retrieve_Simple_Orders()
             */
 
         } 
+        [LusidFeature("F8")]
+        [Test]
+        public void Upsert_And_Delete_Simple_Orders()
+        {
+            var testScope = $"Orders-Filter-TestScope";
+            var order1 = $"Order-{Guid.NewGuid().ToString()}";
+            var orderId1 = new ResourceId(testScope, order1);
+
+            // We want to make a request to upsert an order.
+            var request = new OrderSetRequest(
+                orderRequests: new List<OrderRequest>
+                {
+                    new OrderRequest(
+                        id: orderId1,
+                        quantity: 99,
+                        // These instrument identifiers should all map to the same instrument. If the instance of
+                        // LUSID has the specified instruments registered these identifiers will get resolved to
+                        // an actual internal instrument on upsert; otherwise, they'll be resolved to instrument or
+                        // currency unknown.
+                        instrumentIdentifiers: new Dictionary<string, string>()
+                        {
+                            ["Instrument/default/LusidInstrumentId"] = _instrumentIds.First()
+                        },
+                        // [Experimental] Currently this portfolio doesn't need to exist. As the domain model evolves
+                        // this reference might disappear, or might become a strict reference to an existing portfolio
+                        portfolioId: new ResourceId(testScope, "OrdersTestPortfolio"),
+                        properties: new Dictionary<string, PerpetualProperty>
+                        {
+                            { $"Order/{testScope}/TIF", new PerpetualProperty($"Order/{testScope}/TIF", new PropertyValue("GTC")) },
```

Comment:
where are these properties created or are they assumed to exist (i.e. system properties)